src/main/java/org/broadinstitute/consent/http/db/mapper/DraftSubmissionInterfaceMapper.java

```diff
@@ -0,0 +1,87 @@
+package org.broadinstitute.consent.http.db.mapper;
+
+import java.lang.reflect.InvocationTargetException;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Date;
+import java.util.UUID;
+import org.broadinstitute.consent.http.exceptions.NoMatchingClassException;
+import org.broadinstitute.consent.http.models.DraftInterface;
+import org.broadinstitute.consent.http.models.User;
+import org.jdbi.v3.core.mapper.RowMapper;
+import org.jdbi.v3.core.statement.StatementContext;
+
+public class DraftSubmissionInterfaceMapper implements RowMapper<DraftInterface>, RowMapperHelper {
+
+  @Override
+  public DraftInterface map(ResultSet rs, StatementContext ctx) throws SQLException, NoMatchingClassException {
+    if (!hasColumn(rs, "schema_class")) {
+      throw new NoMatchingClassException("Missing class name.");
+    }
+
+    DraftInterface dsi;
+    String className = rs.getString("schema_class");
+
+    try {
+      dsi = (DraftInterface) Class.forName(className).getDeclaredConstructor().newInstance();
+    } catch (ClassNotFoundException | NoSuchMethodException | InvocationTargetException |
+             InstantiationException | IllegalAccessException classNotFoundException) {
```

Comment:
Name `classNotFoundException` is confusing here as it could be any of these exceptions. 
```suggestion
             InstantiationException | IllegalAccessException e) {
```

Reply:
Renamed.

Reply:
...more like OBE with my next push